README.md

```diff
@@ -122,12 +118,13 @@ of `aicsimageio` while using `napari` please install
 
 ## Performance Considerations
 * **If your image fits into memory and you are not using a distributed cluster:** use
-`AICSImage.data` or `Reader.data` which are generally optimal.
+`AICSImage.data` or `Reader.data` which are generally optimal. You can also use this to
+preload the data before using `get_image_data`.
 * **If your image is too large to fit into memory:** use `AICSImage.get_image_data` to
 get a `numpy` array or `AICSImage.get_image_dask_data` to get a `dask` array for a
 specific chunk of data from the image.
-* **If you are using a distributed cluster:** all functions and properties in the
-library are generally optimal.
+* **If you are using a distributed cluster with more than ~6 workers:** all functions
+and properties in the library are generally optimal.
 * **If you are using a distributed cluster with less than ~6 workers:** use
 `aicsimageio.use_dask(False)`. From our testing, 6 workers is the bare minimum for
```

Comment:
What happens if i set use_dask(False) but then use the aicsimageio dask api?

Reply:
This is a good catch and something that should be changed here / `release-4.0`.

`use_dask` **only** affects the `.data` / `_read_immediate` downstreams. Because of the change to `get_image_data` I don't think we actually need to have `use_dask` anymore interestingly. Because if the API promise is: `dask` in the title means delayed and no `dask` means potentially in memory then they are two nearly entirely separate things? Hard to discuss over typing happy to discuss over call later.

Reply:
As discussed on slack call, this can be resolves and `use_dask` can be entirely removed in 4.0!